app/Nova/Travel.php

```diff
@@ -115,10 +118,109 @@ public function fields(Request $request): array
+                            .' the status on each travel assignment as they are submitted.'
+                        )
+                        ->hideFromIndex(),
+
+                    BooleanGroup::make('Transportation Mode', 'tar_transportation_mode')
+                        ->options(
+                            [
+                                'state_contract_airline' => 'State Contract Airline',
+                                'non_contract_airline' => 'Non-Contract Airline',
+                                'personal_automobile' => 'Personal Automobile',
+                                'rental_vehicle' => 'Rental Vehicle',
+                                'other' => 'Other',
+                            ]
+                        )->help(
+                            'Select all transportation modes that will be used. This will be populated on TAR forms.'
+                        )
+                        ->hideFromIndex(),
+
+                    Text::make('Itinerary', 'tar_itinerary')
+                        ->required()
+                        ->rules('required', 'max:255')
+                        ->help(
+                            'This will be populated on TAR forms.'
+                        )
+                        ->hideFromIndex(),
+
+                    Text::make('Purpose', 'tar_purpose')
+                        ->required()
+                        ->rules('required', 'max:255')
+                        ->help(
+                            'This will be populated on TAR forms.'
+                        )
+                        ->hideFromIndex(),
+
+                    Currency::make('Airfare Cost', 'tar_airfare')
+                        ->required()
+                        ->rules('required', 'integer')
+                        ->min(1)
+                        ->max(1000)
+                        ->help(
+                            'Enter the estimated airfare cost per person in this field.'
+                            .' If you are not traveling by air, enter 0.'
+                        )
+                        ->hideFromIndex(),
+
+                    Currency::make('Lodging Cost', 'tar_lodging')
+                        ->required()
+                        ->rules('required', 'integer')
+                        ->min(1)
+                        ->max(1000)
+                        ->help(
+                            'Enter the estimated lodging cost per person in this field.'
+                            .' If you are not staying overnight, enter 0.'
+                        )
+                        ->hideFromIndex(),
+
+                    Currency::make('Mileage Cost', 'tar_mileage')
+                        ->required()
+                        ->rules('required', 'integer')
+                        ->min(1)
+                        ->max(1000)
+                        ->help(
+                            'Enter the estimated rental vehicle or mileage cost per person in this field.'
```

Comment:
I thought we always put mileage at zero as we don't reimburse mileage.

Reply:
I was thinking rental cars go in this field, but on second thought other transportation is probably more appropriate.